templates/guide.html

```diff
@@ -1,69 +1,54 @@
 {% from "helpers.html" import event_box %}
-<!doctype html>
+<!DOCTYPE html>
 <html>
+  <head>
+    <title>DormCon Guide for {{ api.name }}</title>
+    <meta http-equiv="content-type" content="text/html;charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" href="static/guide-bw.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Noto+Emoji&family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
 
-<head>
-  <title>DormCon Guide for {{ api.name }}</title>
-  <meta http-equiv="content-type" content="text/html;charset=utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" type="text/css" href="static/guide-bw.css">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Noto+Emoji&family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
-</head>
-
-<body>
-  {% include "guide_fm.html" %}
-  <div>
-    <span class="day-title">Residence Hall Tours</span>
-    {% for event in tours %}
-    {{ event_box(event) }}
-    {% endfor %}
-
-    <div class="fpop">
-      <span class="fpop-title">FPOP Week Events</span>
-      During the week before REX begins, FPOPs are happening on campus. If you happen to be around in the afternoon
-      after your FPOP ends, feel free to drop by one of the events below!
-      {% for day in dates.before %}
-      <span class="day-title">{{ day.strftime("%A %D") }}</span>
-      {% for key, events in by_dates.items() %}
-        {% if key == day %}
-          {% for event in events %}
-          {{ event_box(event) }}
-          {% endfor %}
-        {% endif %}
-      {% endfor %}
-      {% endfor %}
-    </div>
-
-    {% for day in dates.rex %}
-    <div class="page">
-      <span class="day-title">{{ day.strftime("%A %D") }}</span>
-      {% for key, events in by_dates.items() %}
-      {% if key == day %}
-      {% for event in events %}
-      {{ event_box(event) }}
-      {% endfor %}
+  <body>
+    {% include "guide_fm.html" %}
+    <div>
+      <span class="day-title">Residence Hall Tours</span>
+      {% for event in tours %} {{ event_box(event) }} {% endfor %} {% if
+      dates.before %}
```

Comment:
Nice use of if-statements to exclude these sections when we have no events before and after!